readme.txt

```diff
@@ -90,6 +90,7 @@ Gateways:
 * cpsms.dk
 * bulksmshyderabad.co.in
 * ozioma.net
+* Bulutfon.com
```

Comment:
Hi,

Can we send the information to hi at veronalabs dot com email? I don't want to share token information publicly. 

Regards.